src/storage/storage.ts

```diff
@@ -496,6 +496,29 @@ export class Storage implements IStorage {
     }
   }
 
+  /**
+   * Check if an utxo is selected as input.
+   * 
+   * @param {IUtxoId} utxo The utxo we want to check if it is selected as input
+   * @returns {Promise<boolean>}
+   */
```

Comment:
I suggest adding an example, so it's easier to remember what is an `IUtxoId` in this context
```suggestion
  /**
   * Check if an utxo is selected as input.
   * 
   * @param {IUtxoId} utxo The utxo we want to check if it is selected as input
   * @returns {Promise<boolean>}
   * @example
   * const isSelected = await isUtxoSelectedAsInput({ txId: 'tx1', index: 0 });
   */
```